### ISSUE TITLE
Panoramic Texture View

### ISSUE THREAD

**Issue by chris-traxler** [pro3d-space/PRo3D#399]
Add the respective panoramic texture view rendered from the HiRISE ortho image as output option.
In case of a priority-rendered photosphere, this one is rendered instead.

**Comment by chris-traxler** [comment]
Also related to #353

**Comment by chris-traxler** [comment]
How large are the panorama texture? The current GPU limit is 32k (in our case the x-axis).

**Comment by chris-traxler** [comment]
The import size is limited by 32k for each coordinate axis. Resampling is done when importing, if necessary.
Alternative: Use array textures.
Further handling of this is done as additional temporary (texture) layer.

**Comment by haraldsteinlechner** [comment]
@gpaa can we get an example panorama for some known location to use it for testing?

**Comment by gpaar** [comment]
Small ones:
0382 3335 (a multispectral one onto the crater rim current traverse)
0397 8420 (Moro rock)


![Image](https://github.com/user-attachments/assets/9682d7fa-faa0-4786-8266-7471a26849fe)
![Image](https://github.com/user-attachments/assets/11ed22da-abe9-4169-9473-7b075eab855c)

Larger one:
1357 9423 (lookout point onto full Jezero traverse)
![Image](https://github.com/user-attachments/assets/7fc8867d-7c1d-4cf4-82b4-a911d4b0f4cf)

All these products are spherical geometry, the Job_xxxx_yyyy-zzz-rad_mosaic.zip, containing meta data in the .json, mainly in the map_transformation entry. Liaise with @ArnoldBauer on units and meaning, he will also re-process until end of this week to provide also geographic and Mars-centered sphere positions. Presently, angles are given in geographic coordinate system with a position presumably only in site frame, the pointer to site is missing and will be added in the newly processed products.

**Comment by haraldsteinlechner** [comment]
This one can be realized by exporting a panorama depth map for a partiulcar viewpoint and use the depth map in the processing pipeline directly. However, it is needed to export the transformation of a photosophere adjustment. 
@gpaar This adjustment could be done using an opc photosophere, right? Or are we talking about "live" projection of a small panorama texture?

**Comment by gpaar** [comment]
Yes the adjustment will be done manually (and in future by horizon matching) on the already existing standard-PRoViP produced Photosphere.

**Comment by chris-traxler** [comment]
Dependent on #518.

**Comment by chris-traxler** [comment]
Clarify if surface nodes export is sufficient to save transformations as mentioned in [#518](https://github.com/pro3d-space/PRo3D/issues/518) .
